CHANGELOG.md

```diff
@@ -140,6 +140,48 @@ The library has been expanded to include the v2 service.
 consider this a breaking change, as Bazel 5.x has been in maintenance mode for
 more than 6 months.
 
+**CMake Proto Libraries**: We no longer compile certain protos, unless the
+corresponding client library is enabled, via `-DGOOGLE_CLOUD_CPP_ENABLE=...`.
+
+This change reduces build times with `cmake` for customers who build with
```

Comment:
I would use CMake (without code style) in this context.  I think we should use `cmake` (with code style) when referring to the command itself, not the system.

Reply:
And remove the *"with CMake"* duplication.  So, *"This change reduces build times for customers who use CMake but who are not using ...."*

Reply:
Done and done.